Userland/Libraries/LibGUI/EmojiInputDialog.gml

```diff
@@ -24,7 +23,7 @@
 
     @GUI::ScrollableContainerWidget {
         name: "scrollable_container"
-        content_widget: @GUI::Widget {
+        @GUI::Widget {
```

Comment:
This doesn't really look valid to me, but I'm far from being an expert in GML. cc @kleinesfilmroellchen